### PR TITLE
Add questions to Scope & Closures, Ch2-5

### PR DESCRIPTION
**Pull request by austintackaberry**

### CODE REVIEW

**Comment by nikrb** [comment]
looks good, except I see some dodgy chars, which seem to be affecting the text position:
screenshot

File: src/components/markdown-renderers/Paragraph.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default class Paragraph extends React.PureComponent {
+  render() {
+    console.log(this.props);
```
Comment (nikrb) [inline src/components/markdown-renderers/Paragraph.js]: should we remove console logs?

**Comment by austintackaberry** [comment]
@nikrb Should be fixed now

**Comment by nikrb** [comment]
I am seeing a black background around the coloured text for question 1 on up and going, is that correct? It just looks a bit odd ... looks good otherwise, have approved.

**Pull request merged**